dbt/models/iasworld/schema/iasworld.sales.yml

```diff
@@ -187,3 +187,7 @@ sources:
                 - saledt
                 - instruno
               config: *unique-conditions
+          - dbt_utils.equal_rowcount:
+              name: iasworld_sales_row_count_by_year_matches_with_mydec
+              compare_model: source('sale', 'mydec')
```

Comment:
**[Nitpick]** I think this test is not actually testing whether the rowcount by year of `iasworld.sales` matches mydec, since `dbt_utils.equal_rowcount` does not group by year, right? If so, I think we either need to change the name of this test to something like `iasworld_sales_row_count_matches_with_mydec` or delete it. It seems like we have tests that group by year defined below, so perhaps the right move is just to delete this block then.

Reply:
@jeancochrane I believe you're correct. I'll work out what the intention here was with @Damonamajor.